dydx-testnet-4/genesis.json

```diff
@@ -0,0 +1,6642 @@
+        {
+          "id": 30,
+          "perpetual_clob_metadata": {
+            "perpetual_id": 30
+          },
+          "quantum_conversion_exponent": -9,
+          "status": "STATUS_ACTIVE",
+          "step_base_quantums": 1000000,
+          "subticks_per_tick": 1000000
+        },
+        {
+          "id": 31,
+          "perpetual_clob_metadata": {
+            "perpetual_id": 31
+          },
+          "quantum_conversion_exponent": -9,
+          "status": "STATUS_ACTIVE",
+          "step_base_quantums": 1000000,
+          "subticks_per_tick": 1000000
+        },
+        {
+          "id": 32,
+          "perpetual_clob_metadata": {
+            "perpetual_id": 32
+          },
+          "quantum_conversion_exponent": -9,
+          "status": "STATUS_ACTIVE",
+          "step_base_quantums": 1000000,
+          "subticks_per_tick": 1000000
+        }
+      ],
+      "equity_tier_limit_config": {
+        "short_term_order_equity_tiers": [
+          {
+            "limit": 0,
+            "usd_tnc_required": "0"
+          },
+          {
+            "limit": 1,
+            "usd_tnc_required": "20000000"
+          },
+          {
+            "limit": 5,
+            "usd_tnc_required": "100000000"
+          },
+          {
+            "limit": 10,
+            "usd_tnc_required": "1000000000"
+          },
+          {
+            "limit": 100,
+            "usd_tnc_required": "10000000000"
+          },
+          {
+            "limit": 1000,
+            "usd_tnc_required": "100000000000"
+          }
+        ],
+        "stateful_order_equity_tiers": [
+          {
+            "limit": 0,
+            "usd_tnc_required": "0"
+          },
+          {
+            "limit": 1,
+            "usd_tnc_required": "20000000"
+          },
+          {
+            "limit": 5,
+            "usd_tnc_required": "100000000"
+          },
+          {
+            "limit": 10,
+            "usd_tnc_required": "1000000000"
+          },
+          {
+            "limit": 100,
+            "usd_tnc_required": "10000000000"
+          },
+          {
+            "limit": 200,
+            "usd_tnc_required": "100000000000"
+          }
+        ]
+      },
+      "liquidations_config": {
+        "fillable_price_config": {
+          "bankruptcy_adjustment_ppm": 1000000,
+          "spread_to_maintenance_margin_ratio_ppm": 1500000
+        },
+        "max_liquidation_fee_ppm": 15000,
+        "position_block_limits": {
+          "max_position_portion_liquidated_ppm": 100000,
+          "min_position_notional_liquidated": 1000
+        },
+        "subaccount_block_limits": {
+          "max_notional_liquidated": 100000000000,
+          "max_quantums_insurance_lost": 1000000000000
+        }
+      }
+    },
+    "crisis": {
+      "constant_fee": {
+        "amount": "1000",
+        "denom": "adv4tnt"
+      }
+    },
+    "delaymsg": {
+      "delayed_messages": [
+        {
+          "block_height": 378000,
```

Comment:
is this value correct for the testnet #4?

Reply:
Yes been using this in testnet-3[ as well](https://github.com/dydxprotocol/networks/blob/13e474ae24891aecef008db7ece42b58de2ad37c/dydx-testnet-3/genesis.json#L1655)

Reply:
does this mean that the fees go live after 72 hours? can we reduce this to 36 hours so we can see the effects go live sooner? `188000` should be around that i think

Reply:
Discussed offline,  will use `57600` which is ~24hours